custom/enikshay/integrations/tests/test_case_properties_changed.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+
+import uuid
+
+from django.test import TestCase, override_settings
+
+from casexml.apps.case.mock import CaseBlock
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.hqcase.utils import submit_case_blocks
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from custom.enikshay.integrations.utils import case_properties_changed
+
+
+@override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
+class TestCasePropertiesChanged(TestCase):
+    def setUp(self):
+        self.domain = 'domain'
+        create_domain(self.domain)
+        case_type = "case"
+        self.case_id = uuid.uuid4().hex
+        submit_case_blocks(CaseBlock(
+            self.case_id, case_type=case_type, create=True).as_string(), self.domain
+        )
+        caseblock1 = CaseBlock(
+            self.case_id,
+            case_type=case_type,
+            update={'property_1': 'updated'},
+        )
+        caseblock2 = CaseBlock(
+            self.case_id,
+            case_type=case_type,
+            update={'property_2': 'updated'},
+        )
+        blocks = [caseblock1.as_string(), caseblock2.as_string()]
+        submit_case_blocks(blocks, self.domain)[1][0]
```

Comment:
what are those indexes doing on the end? looks like you aren't doing anything with the return value.

Reply:
oops, good catch. thanks